styles/styles.css

```
@@ -413,10 +662,33 @@ body {
   color: #ffffff;
 }
 
+.breadcrumbs-item:not(:nth-child(2)):hover {
+  opacity: 0.6;
+}
+
+.breadcrumbs-item:not(:nth-child(2)):focus {
```

```suggestion
.breadcrumbs-item:not(:nth-child(2)):focus {
```
очень сложные и не поддерживаемый селектор. сделай через дополнительный класс